### PR TITLE
Create Juniper kit

### PR DESCRIPTION
**Pull request by keith-smiley-gravwell**
This PR DOES NOT address any issues
Net new kit

### CODE REVIEW

File: juniper/MANIFEST [hunk 1]
```diff
@@ -0,0 +1,168 @@
+{
+	"ID": "io.gravwell.juniper",
+	"Name": "Juniper",
+	"Desc": "",
+	"Readme": "Juniper Readme",
+	"Version": 3,
```
Comment (ashnwade) [inline juniper/MANIFEST:6]: If we are creating this as a new kit, should this be version 1?

**Comment by mike-wade-gravwell** [comment]
1. It needs images for the kit.
2. All the objects need descriptions

Besides those two it LGTM

**Comment by keith-smiley-gravwell** [comment]
> 1. It needs images for the kit.
> 2. It needs a readme
> 3. All the objects need descriptions
> 
> Besides those items it LGTM

- Images are stock for the time being...I've requested permission to use the Juniper logo, but haven't heard back yet
- Added README as README.md as well as in the MANIFEST
- Added descriptions for all the things

**Review by mike-wade-gravwell — APPROVED** [review]
LGTM! Readme looks great.

**Pull request merged**